examples/powertools-examples-large-messages/src/main/java/helloworld/App.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package helloworld;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.lambda.powertools.largemessages.LargeMessages;
+
+/**
+ * Example handler showing how to use LargeMessageProcessor functionally.
+ * This approach gives you more control than the @LargeMessage annotation.
+ */
+public final class App implements RequestHandler<SQSEvent, String> {
+
+    private static final Logger LOG = LogManager.getLogger(App.class);
```

Comment:
Following our documentation, let's use SLF4J here instead of direct calls to Log4j2. Also, we forgot to add the `@Logging` annotation to the `handleRequest` method.

https://docs.aws.amazon.com/powertools/java/latest/core/logging/